### ISSUE TITLE
The return url is just cancatinated with the return params

### ISSUE THREAD

**Issue by emartynov** [paypal/paypal-android#161]
### PayPal Android SDK Version

0.0.10

### Environment

Sandbox

### PayPal dependencies

paypal-payments-sdk = { module = "com.paypal.android:paypal-native-payments", version.ref = "0.0.10" }

### Describe the bug

I'm passing the return url like "schema://paypal" when constructing native checkout client.

After the Paypal checkout is finished, I see my app is notified with URL like `schema://paypalxo?token....`.
So SDK just operates with URL like with string and doesn'tadd an extra slash to URL.

### To reproduce

See describe section

### Expected behavior

The url slash is appended before the return parameters

**Comment by sshropshire** [comment]
@emartynov thanks for your feedback. This is a valid URL according to [RFC 3986](https://www.ietf.org/rfc/rfc3986.txt) (I was able to double check using this [tool](https://0mg.github.io/tools/uri/)). The slash before the return parameters isn't needed. Closing for now.

**Comment by emartynov** [comment]
Did I mention that it is an invalid URL?

Let me rephrase the behaviour and expected behaviour.

Current behaviour:
1. I pass "schema://host"
2. I get back "schema://hostxo?param=value..."

I would expect the following:
1. I get back "schema://host/xo?param=value..."

**Comment by sshropshire** [comment]
@emartynov interesting ok. Let me follow up on this, I see the potential for confusion now.

**Comment by emartynov** [comment]
BTW I fixed it in my codebase by passing "schema://host/" as a return URL.

It is an easy fix on Android - instead of treating the passed URL as a string, convert it to URI and operate on top of it by adding segments, etc.

This would be a nicer UX for users of the library. I assume the root case lies in the javascript SDK. However, your SDK as an intermediary level could make a small change here.

**Comment by emartynov** [comment]
My previous message needs to be corrected. It doesn't matter if I add or not the trailing slash to the return URL.

I still get `xo?....` concatenated without a slash.

**Comment by sshropshire** [comment]
@emartynov is this still an issue as well? I know you said you were able to work around it.

**Comment by emartynov** [comment]
Let me check it

**Comment by devchristina** [comment]
Hi all. Any movement/follow up on this issue? Would love to close if not.
